docs/getting-started.md

```diff
@@ -166,6 +166,24 @@ customization now is the time to do it. Take a look at
 
 Ensure the `region` and `keyName` are set to what you expect.
 
+#### Cluster name
+
+By default the cluster name is set `test1` when generating manifests, `CLUSTER_NAME` environment variable is used in `make manifests` to customize the cluster name.
+
+```bash
+export CLUSTER_NAME="<pony-unicorns>"
```

Comment:
This variable seems not to work. I set it up and `make manifests{,-dev}` still seems to generate manifests for cluster named `test1`. Please let me know if I'm doing something wrong.